usr/share/rear/conf/default.conf

```diff
@@ -740,6 +740,67 @@ OPAL_PBA_DEBUG_PASSWORD=''
 # installed. To test the PBA system on a machine without any Opal 2-compliant disk, set OPAL_PBA_DEBUG_DEVICE_COUNT=1.
 # Used to debug the PBA system.
 OPAL_PBA_DEBUG_DEVICE_COUNT=""
+#
+# Path to linux block device to use as AuthToken container (a small unformatted partition on removable media is ideal)
+# AuthToken may then be generated in-PBA and used as a password replacement,
+# i.e. to unlock disks w/o typing your Opal password each time
+# You may use any of standard /dev/disk/by-whatever/whatever symliks here, 'by-partuuid' should be the most robust one
+# If configured device is detected at the time you unlock your disks with password, PBA will offer you to create a token...
+OPAL_PBA_TKNPATH=""
+#
+# Raw byte offset at which to store AuthToken for current PBA
+# Single container may hold multiple ATs for different PBAs at different offsets
+OPAL_PBA_TKNOFFSET=0
+#
+# AuthToken encryption key
+# The simplest and THE LEAST SECURE option is just set some password here, e.g. 'mYt0kEnpAs$wOrd'
+# 	!!! At least make it unique! It will be stored in cleartext inside PBA image !!!
```

Comment:
Avoid exclamation marks and capitalization of normal words
in description texts because it can be percieved
as if we shouted at our users.

Cf. our SSH_UNPROTECTED_PRIVATE_KEYS and
BACKUP_PROG_CRYPT_KEY description texts
and in general our description texts
about 'secrets' in default.conf

Also avoid words like "just", "simple", "easy", ...
because when a user reads documentation then he is often
already frustrated or even desperate (because normally
users don't read anything unless things don't work)
and then such words could implicitly treat him
as if he was an idiot because such words
implicitly tell things were "just" "simple" "easy" but
in fact disaster recovery is not "just" "simple" "easy", cf.
https://en.opensuse.org/SDB:Disaster_Recovery#Inappropriate_expectations

In general in technical documentation one should
keep it sound neutral and avoid a personal tone.
So neither a military parade ground commanding tone
nor as if you talk to someone personally.